PsfLauncher/main.cpp

```diff
@@ -146,7 +145,22 @@ int launcher_main(PCWSTR args, int cmdShow) noexcept try
         LogString("Working Directory: ", currentDirectory.c_str());
 
         //THROW_IF_FAILED(StartProcess(exePath.c_str(), (L"\"" + exePath.filename().native() + L"\" " + exeArgString + L" " + args).data(), (packageRoot / dirStr).c_str(), cmdShow, INFINITE));
-        HRESULT hr = StartProcess(exePath.c_str(), (L"\"" + exePath.filename().native() + L"\" " + exeArgString + L" " + args).data(), (packageRoot / dirStr).c_str(), cmdShow, INFINITE);
+        HRESULT hr = ERROR_SUCCESS;
+        bool createProcessesInAppContext = false;
+        auto createProcessesInAppContextPtr = appConfig->try_get("InPackageContext");
+        ProcThreadAttributeList m_AttributeList;
+        if (createProcessesInAppContextPtr)
+        {
+            createProcessesInAppContext = createProcessesInAppContextPtr->as_boolean().get();
+        }
+        if (createProcessesInAppContext)
+        {
+            hr = StartProcess(exePath.c_str(), (L"\"" + exePath.filename().native() + L"\" " + exeArgString + L" " + args).data(), (packageRoot / dirStr).c_str(), cmdShow, INFINITE, m_AttributeList.get());

```

Comment:
how is m_AttributeList setting the BreakAway flag?

Reply:
Through its constructor